sfdc/src/main/java/com/jalasoft/sfdc/ui/pages/priceBook/PriceBookFormLight.java

```diff
@@ -0,0 +1,28 @@
+package com.jalasoft.sfdc.ui.pages.priceBook;
+
+import com.jalasoft.sfdc.ui.pages.DetailPage;
+import com.jalasoft.sfdc.ui.pages.account.DetailAccountPageClassic;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class PriceBookFormLight extends PriceBookForm{
+    @FindBy(xpath = "//button[@title='Save']")
+    WebElement saveButton;
+    @FindBy(xpath = "(//div[@class='uiInput uiInputText uiInput--default uiInput--input']/child::input)[1]")
+    private WebElement inputName;
+
+    @FindBy(xpath = "(//div[@class='uiInput uiInputText uiInput--default uiInput--input']/child::input)[2]")
+    private WebElement inputDescription;
+    @FindBy(xpath = "//div[@class='uiInput uiInputCheckbox uiInput--default uiInput--checkbox']/child::input")
+    private WebElement checkBox;
+
+    public DetailPage setFields(String name, String description, String active) {
```

Comment:
based on the implementation this method should be named createPriceBook, not only setFIelds